packages/components/CHANGELOG.md

```diff
@@ -4,14 +4,13 @@
 
 ### New Feature
 
--   Add a new `ProgressBar` component. ([#53030](https://github.com/WordPress/gutenberg/pull/53030)).
-
```

Comment:
Looks like this removal might be accidental via a rebase?

Reply:
Oh dang, thanks for picking that up. Yeah, dodgy rebase 🤦🏻‍♂️